pkg/reconciler/awssqssource/awssqssource.go

```diff
@@ -38,12 +39,53 @@ import (
 	"k8s.io/client-go/rest"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
 const (
+	// controllerAgentName is the string used by this controller to
+	// identify itself when creating events.
+	controllerAgentName = "aws-sqs-source-controller"
+
+	// awsSqsEnabledEnvVar is used to determine if the AWS SQS Source's
+	// controller should run.  It will only run if the environment
+	// variable is defined and has the value 'true'.
+	awsSqsEnabledEnvVar = "ENABLE_AWSSQS_SOURCE"
+
+	// raImageEnvVar is the name of the environment variable that
+	// contains the receive adapter's image. It must be defined.
+	raImageEnvVar = "AWSSQS_RA_IMAGE"
+
 	finalizerName = controllerAgentName
 )
 
+// Add creates a new AwsSqsSource Controller and adds it to the Manager with
+// default RBAC. The Manager will set fields on the Controller and Start it when
```

Comment:
ditto here.

Reply:
Add clarity to the comment for awssqusource.Add()

Reply:
https://github.com/knative/eventing-sources/issues/203